lib/rails_admin/abstract_model.rb

```diff
@@ -153,13 +153,17 @@ def build_statement_for_type
       def build_statement_for_integer_decimal_or_float
         case @value
         when Array then
-          val, range_begin, range_end = *@value.collect do |v|
+          collect_proc = Proc.new do |v|
```

Comment:
since it is safe to assume, that everybody already using Ruby +1.9 we can use shorthand:

```ruby
  proc do |v|
    [...]
  end
```